### PR TITLE
update container-tests to use newest testing farm action

### PR DESCRIPTION
**Pull request by zmiklank**
This can be merged only when TFGA v1.2.9 is released (with this change integrated: https://github.com/sclorg/testing-farm-as-github-action/pull/40)

### CODE REVIEW

**Comment by zmiklank** [comment]
The TF-as-GithubAction is in the required state, so this can be merged (after a review).

File: .github/workflows/container-tests.yml
Comment (phracek) [inline .github/workflows/container-tests.yml:23]: This is not needed, I guess. The default value is public.

File: .github/workflows/container-tests.yml
Comment (phracek) [inline .github/workflows/container-tests.yml:70]:
```suggestion
        uses: sclorg/testing-farm-as-github-action@v1
```

**Review by phracek — CHANGES_REQUESTED** [review]
I guess, tf_scope: 'public' is not needed. It is set by default.

File: .github/workflows/container-tests.yml
Comment (zmiklank) [reply .github/workflows/container-tests.yml:23]: Yes, but we take `tf_scope` input from the matrix. And in the matrix, there is no default. So IMHO empty value would be passed to the action, or it will fail.

Comment (phracek) [reply .github/workflows/container-tests.yml:23]: Undefined `tf_scope` in the matrix works properly. See https://github.com/sclorg/s2i-base-container/pull/209.
And the reason is simple. Only if `tf_scope` is defined as `private` then `osci...` is used otherwise `public` URL.

**Review by phracek — APPROVED** [review]
LGTM. As soon as tests are passing, you can merge it.

File: .github/workflows/container-tests.yml
Comment (zmiklank) [reply .github/workflows/container-tests.yml:72]: Sorry, I do not understand, why the `tf_scope` should be removed also from actions inputs. Wouldn't then always be used the Public Testing Farm?

**Pull request merged**
